gulpfile.js

```diff
@@ -13,8 +13,9 @@ const WEBFONTS_SRC = "./fonts/webfonts";
 const WEBFONTS_DEST = "./site/assets/fonts";
 
 function copyWebfonts() {
-  return src(`${WEBFONTS_SRC}/**/**`)
-    .pipe(dest(WEBFONTS_DEST));
+  return src(`${WEBFONTS_SRC}/**/**`, {
+    encoding: false,
+  }).pipe(dest(WEBFONTS_DEST));
```

Comment:
Added to address the issue fixed in https://github.com/uswds/uswds-compile/pull/96 from a solution noted in https://github.com/gulpjs/gulp/issues/2803